Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -56,19 +72,37 @@ public class ConnectionPoolConnection: Connection {
         }
     }
     
-    /// Close the connection to the database.
+    /**
+     Close the connection to the database.
+     ### Usage Example: ###
+     In this example, The closeConnection() function is called on a `ConnectionPoolConnection` instance called "connection" to release it from it's `ConnectionPool` instance.
```

Comment:
"In this example, The closeConnection() function is called on a `ConnectionPoolConnection` instance called "connection" to release it from it's `ConnectionPool` instance."
how about 
"In this example, the `closeConnection()` function is called on a `ConnectionPoolConnection` instance to release it from its `ConnectionPool` instance."